simulation/__main__.py

```diff
@@ -52,6 +52,12 @@ def restrict_between_0_and_1(input_value: float):
         help="A value between 0 and 1 which suppresses the likelihood of a household choosing a given heating system (the higher the value, the lower the likelihood)",
     )
 
+    parser.add_argument(
+        "--intervention",
+        choices=["rhi"],
+        type=str,
```

Comment:
What do you think about making this a `type=InterventionType` and using a trick of `choices=list(InterventionType)` to convert it to the enum type right off the bat, rather than converting it inside the agent class?

Reference: https://stackoverflow.com/questions/43968006/support-for-enum-arguments-in-argparse/55500795

Reply:
I'm reluctant to over-engineer the command line args for now, given we might want to fundamentally revisit the way large batches of parameters get passed to a model